test/pages/Inventory.page.js

```diff
@@ -3,15 +3,65 @@ const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
+    selectedItemParameters = []; 
+
     get headerTitle() { return $('.title'); }
 
     get inventoryItems() { return $$('.inventory_item'); }
 
     get addItemToCartBtns() { return $$('[id^="add-to-cart"]'); }
 
+    get sortItemsBtn() { return $('.select_container'); }
+
+    get sortingOption() { return $('option[value="hilo"]'); }
+
+    get currentSortingOption() { return $('.active_option'); }
+
+    get itemsPrice() { return $$('.inventory_item_price'); }
+
+    get cartBadge() { return $('.shopping_cart_badge'); }
+
     async addItemToCartById(id) {
-        await this.addItemToCartBtns[id].click();
+        const selectedItem = this.addItemToCartBtns[id];
+        const card = await selectedItem.parentElement().parentElement();
+        this.selectedItemParameters.push({
```

Comment:
It's better not to store data in a page object, but rather have a method that will get the data. Then store that data in the test itself. Otherwise, you might face strange behaviour once you start running tests in parallel